docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -69,3 +69,78 @@ If the DNS record exists it returns the answer and sets query status to `NO_ERRO
 #### System Requirements for Mode `run-host`
 
 `ziti-edge-tunnel run-host` does not require elevated privileges or the above device or socket, only network egress to the servers for which it is hosting Services.
+
+## Multi-Factor Authentication
+
+You may use multi-factor authentication (MFA) with `ziti-edge-tunnel` by completing the following workflow to register a time-based one-time password (TOTP) token generator with the OpenZiti Controller.
```

Comment:
I think we want to avoid any if, conditional, or optional type language. Instead, these should be focused on education and information, just reciting facts.  For example, you should just tell the user what we support:

OpenZiti supports time-based one time passwords (TOTP) {insert link to totp} as and additional factor for authentication (MFA) {insert link to wikipedia etc}.